rapid_pro_tools/rapid_pro_client.py

```diff
@@ -277,6 +277,55 @@ def send_message_to_urn(self, message, target_urn, interrupt=False):
         log.info(f"Message send request created with broadcast id {response.id}")
         return response.id
 
+    def send_message_to_urns(self, message, target_urns, interrupt=False):
+        """
+        Sends a message to URNs.
+
+        :param message: Text of the message to send.
+        :type message: str
+        :param target_urn: URNs to send the message to.
+        :type target_urn: str
+        :param interrupt: Whether to interrupt the target_urn from flows before sending the message.
+        :type interrupt: bool
+        :return: Ids of the Rapid Pro broadcasts created for this send request.
+                 These ids may be used to check on the status of the broadcast by making further requests to Rapid Pro.
```

Comment:
```suggestion
                 These ids may be used to check on the status of the broadcast by making further requests to Rapid Pro.
                 e.g. using get_broadcast_for_broadcast_id.
```
Might be helpful to point users to how they can do this, I assume I missed this before